configs/tdb_external_kvstore_with_qspif.json

```diff
@@ -0,0 +1,123 @@
+                    "- KVStore is on an external QSPI Flash block device",
+                    "The flash regions are as follows:",
+                    "1. Bootloader - 32KiB from the beginning of flash",
+                    "2. KVSTORE - 2 flash sectors immediately following the bootloader",
+                    "3. Active App Metadata Header - (1KiB/2KiB) from the end of KVSTORE",
+                    "4. Active App - From end of header to the end of flash",
+                    "+--------------------------+",
+                    "|                          |",
+                    "|                          |",
+                    "|                          |",
+                    "|        Active App        |",
+                    "|                          |",
+                    "|                          |",
+                    "|                          |",
+                    "+--------------------------+ <-+ mbed-bootloader.application-start-address",
+                    "|Active App Metadata Header|",
+                    "+--------------------------+ <-+ update-client.application-details",
+                    "|                          |",
+                    "|         KVSTORE          |",
+                    "|                          |",
+                    "+--------------------------+ <-+ storage_tdb_internal.internal_base_address",
+                    "|                          |",
+                    "|        Bootloader        |",
+                    "|                          |",
+                    "+--------------------------+ <-+ 0",
+                    "Firmware Candidate Storage is on external QSPIF device at an offset of 0 MiB.",
+                    "The storage size is the size of the maximum application size (i.e. Active",
+                    "Application region size) multiplied by the number of storage-locations.",
+                    "(storage-locations is set to 1 in this configuration)",
+                    "+--------------------------+<-+ End of QSPIF block device",
+                    "|                          |",
+                    "+--------------------------+<-+ update-client.storage-size + update-client.storage-address",
+                    "|                          |",
+                    "+--------------------------+",
+                    "|                          |",
+                    "|   Firmware Candidate 0   |",
+                    "|                          |",
+                    "+--------------------------+",
+                    "|   Firmware Candidate 0   |",
+                    "|     Metadata Header      |",
+                    "+--------------------------+ <-+ update-client.storage-address (1024 * 1024)",
+                    "|                          |",
+                    "|     External KVStore     |",
+                    "|      Size: 1MB           |",
```

Comment:
cosmetic, but this image has old numbers, should be 128 KiB

Reply:
Fixed